crates/modelardb_common/src/metadata/mod.rs

```diff
@@ -248,9 +257,13 @@ mod tests {
             .unwrap();
 
         let table_name = "table_name";
-        save_table_metadata(&metadata_database_pool, table_name.to_string())
-            .await
-            .unwrap();
+        save_table_metadata(
```

Comment:
There are some issues related to this, for example if you create a table with a name that is a valid folder name but invalid in a particular RDBMS you would create the `empty_for_schema.parquet` file but fail when saving the metadata. 

It might be best to add an explicit check for invalid characters and then maybe test that this check works with an integration test of the `CommandStatementUpdate` action. 

Since we create folders that uses the table name when creating normal tables and when saving compressed data it might also be good to look into OS specific requirements when we do this check of the table name.

Since this is not relevant to this PR, I have added a [work item](https://dev.azure.com/ModelarData/ModelarDB%20Implementation/_workitems/edit/774) to look into this issue later.